app/app.go

```diff
@@ -589,6 +590,12 @@ func NewApp(
 		panic(err)
 	}
 
+	if maxSize := os.Getenv("MAX_WASM_SIZE"); maxSize != "" {
```

Comment:
What's the idea behind the compile time parameter? Just wondering why we would ever want that when the max wasm size is a chain parameter.

Reply:
Perhaps just for testing purpose?

Reply:
Because they might need to be used by `ValidateBasic()` or `Validate()` which is stateless.

Reply:
But yeah if it was changed via governance afterwards, the binary wouldn't be able to pick that up without accessing state. For testing purposes for sure.